### DISCUSSION TITLE
SamplesApp UWP some issue

### DISCUSSION THREAD

**Discussion by workgroupengineering** [unoplatform/uno#8592] [Q&A]
While developing the PR I encountered several problems running SamplesApp in UWP.

My question is if I have to create a PR for each of them or do I create a cumulative?

Here are some of the errors found:

- WinUIDeterminateProgressRing System.NullReferenceException
- UITests.Microsoft_UI_Xaml_Controls.TabViewTests.TabViewPage.TabViewPage_Loaded: System.ArgumentException
- Failed to assign to property 'Windows.UI.Xaml.Media.Animation.ColorKeyFrame.Value' because the type 'Windows.UI.Xaml.Media.SolidColorBrush' cannot be assigned to the type 'Windows.UI.Color'. [Line: 17 Position: 6]
NativeDefaultCommandBar 
-"AppBar_KeyBoard" Failed to assign to property 'Windows.UI.Xaml.Media.Animation.ColorKeyFrame.Value' because the type 'Windows.UI.Xaml.Media.SolidColorBrush' cannot be assigned to the type 'Windows.UI.Color'. [Line: 17 Position: 6]
CommandBar_Xaml_Automated  Cannot find a Resource with the Name/Key NativeDefaultCommandBar [Line: 18 Position: 46]
- "FrameworkElement_Sizes" : Cannot find a Resource with the Name/Key XamlCommandBar [Line: 56 Position: 72]
- ...

more of the error depend do not find resource like XamlDefaultSlider and XamlCommandBar  NativeDefaultCommandBar

**Comment by jeromelaban** [comment]
You can create a cumulative one to fix all the issues, that'll do. Thanks very much!
